playwright/e2e/right-panel/notification-panel.spec.ts

```diff
@@ -22,14 +22,14 @@ const NAME = "Alice";
 test.describe("NotificationPanel", () => {
     test.use({
         displayName: NAME,
+        enableLabFeatures: ["feature_notifications"],
```

Comment:
Personally I think `labsFlags` would be a better name, that way it reads `test use(s) labsFlags X, Y, Z`